mix.exs

```diff
@@ -33,17 +33,16 @@ defmodule ExImageInfo.Mixfile do
   defp deps do
     [
       {:excoveralls, "~> 0.18", only: :test},
-      {:ex_doc, "~> 0.30", only: :dev},
+      {:ex_doc, "~> 0.37", only: :dev},
```

Comment:
Not really required these 2 version updates, as mix.lock is committed, but well, since all is for dev/test, updating it here as well.